PocketKit/Sources/PocketKit/String+Localization.swift

```diff
@@ -2,6 +2,10 @@ import Foundation
 
 extension String {
     func localized(withComment: String = "") -> String {
-        return NSLocalizedString(self, tableName: nil, bundle: Bundle.main, value: "Localizable string not found!", comment: withComment)
+        return NSLocalizedString(self, tableName: nil, bundle: .module, value: "Localizable string not found!", comment: withComment)
+    }
+
+    func localized(_ withArguments: CVarArg..., comment: String = "") -> String {
```

Comment:
in which case do we use this vs the other `localized` function

Reply:
If you would like to pass an argument:
EX: %d min read".localized(timeToRead)

Reply:
thanks for the clarification!